messages/cardano.proto

```diff
@@ -64,11 +64,24 @@ message CardanoSignTransactionRequest {
     uint32 prev_out_index = 3;
   }
 
+  // https://github.com/input-output-hk/cardano-ledger/blob/d0aa86ded0b973b09b629e5aa62aa1e71364d088/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L358
+  message AssetGroup {
+    bytes policy_id = 1;
+
+    message Token {
+      bytes asset_name = 1;
+      uint64 value = 2;
```

Comment:
would appreciate a comment clarifying this is a count of tokens, not an ADA value or a value of tokens in ADA or some other unit.